src/xhk.c

```diff
@@ -271,6 +271,9 @@ int mirror_key(int keycode)
     case KEY_CAPS:
         keycode = KEY_ENTER;
         break;
+    case KEY_GRAVE:
```

Comment:
What about the other direction - KEY_MINUS -> KEY_GRAVE?

Perhaps this should be converted to a table which is searched to save duplication, and make the mirroring explicit (but I don't think that's necessary for this patch)

Reply:
You are right, it make perfect sense to add pair 'minus to grave' mapping